src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```diff
@@ -31,14 +32,19 @@ public ShoppingCartController(ShoppingCartService shoppingCartService,
     }
 
     @PutMapping("/movie-sessions")
-    public void addToCart(@RequestParam Long userId, @RequestParam Long movieSessionId) {
+    public void addToCart(Authentication authentication, @RequestParam Long movieSessionId) {
```

Comment:
lets swap parameters

Reply:
Do you mean authentication first? It's been already like that

Reply:
That ur block of code, not mine ) @RequsetParam should go first, thats what I meant)